helpers/helpers.html

```diff
@@ -85,19 +85,20 @@
         var deltaScrollTop = scrollTop - currentScrollTop;
         var deltaScrollLeft = scrollLeft - currentScrollLeft;
         var duration = 300;
-
-        (function updateFrame() {
+        var updateFrame = (function updateFrame() {
```

Comment:
@keanulee I could do that, but in this case in particular making the distinction doesn't seem to provide any benefit.  wdyt? I guess arrow functions fix cases like this, but we don't use them yet.